docs/proposals/network-policy.md

```diff
@@ -0,0 +1,182 @@
+# Native API Support for Network Policy
+#### Abstract
+A proposal for adding the API primitives to describe flexible network policy.
+
+Network policy is useful for a number of reasons, including:
+- Building secure micro-service based applications.
+- Enforcing multi-tenancy without requiring isolated L2 networks.
```

Comment:
Perhaps leave out "without..."; the isolation policy can be rendered equally well as iptables rules or L2 networks.